src/app/workspace/viewport/viewport.component.ts

```diff
@@ -25,27 +27,9 @@ import { TimingFileType } from '../../import/timing-file-type.enum';
 export class ViewportComponent implements AfterViewInit, OnDestroy {
 
     /**
-     * Color sets for multiple timings.
+     * Maximum difference allowed between 2 timing states positions before being considered an error
```

Comment:
```suggestion
     * Maximum difference allowed between 2 timing states positions before being considered a significant deviation
```

I prefer this more because an Error might make it seem that it is an error within our program. However, we wish to indicate that there is a difference between expected and real (which might potentially be caused by an error in the measured system.)

Reply:
Addressed